tasks/licenses_csv_report.js

```diff
@@ -0,0 +1,72 @@
+import { writeFileSync } from 'fs';
+import { getInstalledPackages } from './lib';
+
+import {
+  isNull,
+  isUndefined
+} from 'lodash';
+
+const allDoubleQuoteRE = /"/g;
+
+function escapeValue(value) {
+  if (isNull(value)) {
+    return;
+  }
+
+  return `"${value.replace(allDoubleQuoteRE, '""')}"`;
+}
+
+function formatCsvValues(fields, values) {
+  return fields.map((field) => {
+    const value = values[field];
+
+    if (isNull(value) || isUndefined(value)) {
+      return null;
+    }
+
+    return value.toString();
+  })
+    .map(escapeValue)
+    .join(',');
+}
+
+export default function licensesCSVReport(grunt) {
+  grunt.registerTask('licenses:csv_report', 'Report of 3rd party dependencies', async function () {
+    const fields = ['name', 'version', 'revision', 'url', 'license', 'copyright'];
+    const done = this.async();
+
+    try {
+      const options = grunt.config('licenses', { overrides: {} });
+      const file = grunt.option('out');
```

Comment:
is it possible to name the option `csv` in order to have `./node_modules/.bin/grunt licenses:csv_report --csv="/foo/licenses.csv" `?